__tests__/integration.js

```diff
@@ -0,0 +1,391 @@
ZB6czYl9f+rTA+ebcQS7TYLnqKRoC29dadjdPvB/wWXborkQBeetFug4mXoqtHcf5KgioxqgrC/bqbcdljBQOWBdrfZ+21+awsKEAeF/jKPoqislBHzRII+4M32kZOgFwUp4VDjaW6GmhuttMZ3YvuziYSlBHwwnb7BOJLnbGUlRW4ttm4Wpg2KH5OSFCKm3zjbRDE9bY4l0AQWLWCrqmZWLvOTqvyKI6iSFAUPclv7GiJKnMRDNot1dtMgtKLR269yTbR09KltEiCWblmhZ1Sva1KpmMhQSmLnh6g1I7QLdWzB1ElURQJqlge/WAFeOETtjiXVSsotaNUT98oigSlefS0zRbnUVWpfJwdoX+qZ48/JI3o7ixq7nISVPGC+owtzmMFpXaWgWWBlla7nE1RaR4Jqhi2b/wUJMn6c6pUVVJh3HKxRyMQDNnhTO4opssBCaqo6EncbIvzaG2ha2lF7FOLKvg5IkEVyiO3XgNBuM3y51G7mIazUBRFgrIdgmiPN3dLl9K1pCjKcNWiu7ODBKWuoD5r+XOg2pP1qauzyxu9gqIoElRh6d1NEMVrLX8e9dStwPKwLNDYZIczKai7AQmqsOjpC7a4sanfkz2wRxeRaCF9okhQhcALf0LRE2EagkG7jNEjQamS3tmh79PVDXQt7YRD0jwSVP707i7rB9MRIBKma2kn6uockeaRoPILapPlz4FqT/aDZe0iKRKUYh79YDkkyfpPd20tXUtbRlG2WMl6BQlKKX/34hj83iawnqcsew6BIPUcpzTPjMQgrz78HhJUaZIawaM/vw9+781wuw5YL3qihRBsneZZ823e0wBaC1kandbFK5btGz+FDP+4Zd7srVvrLElZfV28Yvn9ceD4casc7T4A29HV01voL5CgFEdWH9gBXvgiJMncr8c2/YmzrovTBGWNhT7PZMW0q9hfpBRPKY/+fBt87LvhcT9j2mOMRug62R3zp3iPZNO5XUp+mSIoddK+28AL20230svSpcB1zc66Fk6LoAA5ghocNNtR/RTAVnT19JeyERKUmnxt4xfB8f/DNPWp1hbn9YFyoqCOHAHePGWao8mKqVeNjVGKp2ow+/K34ffeANbzdXOkeFG6Jk6gvNwMRxED8Gfo6mlVS04kKC2QuyVsg9+7Am73C4YeCw1vIUHpwzcBNCmtM1GKZyTbN94GTnhS9/mkqiqBtvXOa28npngA8NyzRux1H4DNpdaZSFBmQK5P7dCtW0LtYrkPFAnKGTz/PJCY1GtvZ7Ji6tV6R5Ti6YVcn9Jv2Ax1MXAWoaAee4kB+Aq6epr0kBMJSm8uD5tZofmwGVrS3FlENa9DPQ25zrRTz9Py0JU1It176U0Abdi+8TZk+P+nSbeEMEVQjsKr2R+kfZC7DfQZcVokKCPZ/vILAOrxdx/4G/DCw6YfNkM4iTNZMe028iAoxTMDj/78H+D3Nqk6bMbrpXZ1ElWqDXmRp0GRh6fsNvq06C2e2Xjk1pvA8d8sediM0wYJ53DqWzx1Bg0/DXlQb79ZTotSPLPxtZd+A7k+Za1pXQgrU/Q0KJTiOZ3tL//r1LAZholTgxAakBue0mFGOZGgzE5u2IzZp3UhLFlMgEbDUyjFc17a9yaAT2L7xu+acloXwkqoMg0KCYqYK+17AcALug+bIeyAqtOgUIpHLBCcTw2b+To1BpGH3PCUVqvJiQRlZcw0rQthVnLToOy06glQimd15PrUhwyb1oUwI5pPg0KCIopDrk81Tw2bSSbDtGCn49BtGhS9oJ7kduTRD1bgXe/6d9TXv8dx5+7EnuQcJ+H48R/g09/6rN1OjQRlZ565vwPl5T9FWcg5Uxs4TVAXL76F0ZGN+NMnz9jx9EhQTuC5rV9FZeU/wOt1k6BswtjYGAYv/Rfc8fiP7HyaJCgn8e8P/Rrl5W3weBgSlEVJJnlcvPg0PvIPf+GEW5YE5TR+fP9SBEM/x6LyJhKUheA4CRcvvoKJ+L12TedIUMRlnv3yZxGJfst29Sk7CurixbcwNrYZH3viV067TUlQTucnDzyNReWfsU19yk6CiseTuHTxb7Dp6zudenuSoAg57fP5/tkW9Sk7CIrjJLzzznedUmciQRGF8cz9HQiF/snS9SmrC+rcuX1OqzORoIjieG7rVxEtfwTBgJ8EpRNDQxcwPPxxJ9aZSFCEMn7ywNOorPicpdI+qwkqmeQx8M5DTq4zkaAI5eTqU1VVG0hQKsJxEs6f/yk+9Pcfo5uMBEWUyjP3dyAS/hEikRoSVImcH+hDLHYn1ZlIUITamH3YjJkF5ZDhKSQownh2P7gXFYs+Yrr6lBkFlUzyOH/+cdz+2F/TjUOCIvTix/cvRSDw76iouI4ENQe54Smdj3bQzUKCIozCTMNmzCIom0+DQoIirMe//fcnULHoy4bWp4wW1NjYGIaHHqFuAyQowqwYOa2LUYJy2DQoJCjC2hg1rYvegnLoNCgkKMIePLf1qwhH/odu9Sk9BUXDU0hQhE3Qa1oXPQRF06CQoAibpn1aT+uipaCozkSCIhwkqkjkZtUjKi0ERWIiQREOTv1CoTtVq1GpKaihoQuIje2gVI4ERTidZ+7vQDDwv1FWdm1JUVWpgorHkxgdfQHJ5BNU/CZBEcRsnv3yZ+FhP4Vg8H1FR1bFCorjJMRiFzEx8TOk07tISiQoglAmLI+7EV7f1fD7wvMW2RcSVDLJI52eQDLRj3TmMLjMizSzAAmKILRLC92um2Z8z+0W4HYLM77HC4coMiJBEQRBqIqLmoAgCBIUQRAECYogCBIUQRAECYogCBIUQRAECYogCIIERRAECYogCIIERRAECYogCIIERRAEQYIiCIIERRAEQYIiCIIERRAEQYIiCIIgQREEQYIiCIIgQREEQYIiCIIgQREEQZCgCIIgQREEQZCgCIIgQREEQZCgCIIgSFAEQZCgCIIgSFAEQZCgCIIgtOf/A7sWdfziDxE6AAAAAElFTkSuQmCC",
```

Comment:
This is the reason this file is originally hidden due to being too large. It's the base64 encoded version of https://github.com/awslabs/aws-serverless-express/blob/master/example/sam-logo.png. A few lines down we're creating a new instance of a server which has a directive to base64 encode the response when it encounters image/* content types.

Reply:
Question. If it is the based64 encoded string for a known file, why not just generate it instead of hardcoding it here?

Reply:
Good callout! I'll do that.

Reply:
Or store it in a txt file